web-app/src/components/UserAvatar.vue

```diff
@@ -18,5 +19,14 @@ interface AvatarProps extends Pick<QAvatar, 'size'> {
 export default class UserAvatar extends Vue {
   @Prop({ type: Object, required: true }) readonly user!: User;
   @Prop({ type: Object, required: false }) readonly avatarProps!: AvatarProps;
+
+  private userPhoto = 'https://cdn.quasar.dev/img/boy-avatar.png';
+
+  private async mounted() {
+    const response = await api.organization.getUserPhoto(this.user.id);
+    if (response !== 'https://cdn.quasar.dev/img/boy-avatar.png') {
+      this.userPhoto = response;
+    }
+  }
```

Comment:
What do you think about leave `this.userPhoto` as empty string and assign response directly to `this.userPhoto`(optionally adding the logic I mentioned above)? `response` always return image, otherwise e.q. we can have same connection problem with server.

Additionally, we will control the default image only from the server side and `UserAvatar` component won't make redundant requests for `boy-avatar.png`